Userland/Libraries/LibGUI/TextEditor.h

```diff
@@ -389,6 +390,7 @@ class TextEditor
     void on_search_results(GUI::TextRange current, Vector<GUI::TextRange> all_results);
 
     static constexpr auto search_results_span_collection_index = 1;
+    static constexpr auto highlight_selected_text_span_collection_index = 1;
```

Comment:
This needs to be a different number than `search_results_span_collection_index`. (Probably 2.) Otherwise it still conflicts with the search results.